File: app/src/main/java/com/example/javakotlin/javakotlinoverview/java/Enumeration.java
package com.example.javakotlin.javakotlinoverview.java;

enum Department {
    IT, SALES, MARKETING, FINANCE
}

public class Enumeration {
    public static void main(String args[]) {
        Department dept;
        dept = Department.MARKETING;
        System.out.println("This is "+dept+ " team.");
    }
}

File: app/src/main/java/com/example/javakotlin/javakotlinoverview/kotlin/Enumeration.kt
package com.example.javakotlin.javakotlinoverview.kotlin

enum class DEPARMTMENT {
    IT, SALES, MARKETING, FINANCE
}

class Enumeration {
    fun main() {
        var dept = 1
        for (dept in DEPARMTMENT.values()) {
            println("${dept.ordinal} = ${dept.name}")
        }
    }
}

